data/jobs.txt

```diff
@@ -1483,6 +1483,169 @@ mission "Core Pirate Attack"
 		payment 200000
 		dialog "The government of <planet> pays you <payment> for helping to drive off the pirates."
 
+mission "Renegade Drones 1"
+	name "Neutralize broken Drone"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drone."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 20
+		"combat rating" > 200
+	clearance
+	source "Pilot"
+	destination "Pilot"
+	on offer
+		conversation
+			`Walking past a group of Navy soldiers, you hear one shout out: "Not again, another drone seems to have a malfunction and attacks ships in orbit. Who's turn is it to shoot it down?"`
+			`Nobody answers.`
+			`What do you want to do?`
```

Comment:
All lines of conversation after the first should be tabbed in once. Example:
```
`Walking past a group of Navy soldiers, you hear one shout out: "Not again, another drone seems to have a malfunction and attacks ships in orbit. Who's turn is it to shoot it down?"`
`	Nobody answers.`
```

Also, why have these two small lines in a row? I think you should trash line 1504 and move line 1503 to the end of 1502.